docs/scenarios/stateless/README.md

```diff
@@ -0,0 +1,192 @@
+## Migrating a *stateless* OpenShift app with mig-controller
+
+This scenario walks through Migration of a stateless OpenShift app, meaning that the app has no Persistent Volumes (PVs) attached.
+
+---
+
+### 1. Prerequisites
+
+Referring to the getting started [README.md](https://github.com/fusor/mig-controller/blob/master/README.md), you'll first need to deploy mig-controller and Velero, and then create the following 'Mig' resources on the cluster where mig-controller is running to prepare for Migration:
+
+- `MigCluster` resources for the _source_ and _destination_ clusters
+- `Cluster` resource for any _remote_ clusters (e.g. clusters the controller will connect to remotely, there will be at least one of these)
+- `MigStorage` providing information on how to store resource YAML in transit between clusters 
+ 
+---
+
+### 2. Deploying a stateless sample app (nginx)
+
+On the _source_ cluster (where you will migrate your application _from_), create the contents of the `nginx-deployment.yaml` manifest. 
+
+```bash
+# Login to the migration 'source cluster'
+$ oc login https://my-source-cluster:8443
+
+# Create resources defined in 'nginx-deployment.yaml'
+$ oc create -f nginx-deployment.yaml 
+
+namespace/nginx-example created
+deployment.apps/nginx-deployment created
+service/my-nginx created
+route.route.openshift.io/my-nginx created
+```
+
+In a few seconds, you should see nginx pods start running.
+
+```bash
+# Making sure our stateless app is running
+$ oc get pods -n nginx-example
+NAME                                READY     STATUS    RESTARTS   AGE
+nginx-deployment-55b5c6f96c-bs5qb   1/1       Running   0          11s
+nginx-deployment-55b5c6f96c-rp4p2   1/1       Running   0          11s
+```
+
+We can check that our nginx pods are serving their starter webpage successfully before continuing.
+
+```bash
+# Get the stateless app route host/port
+$ oc get route -n nginx-example
+NAME       HOST/PORT
+my-nginx   my-nginx-nginx-example.apps.my-source-cluster.example.com 
+
+# Verify the nginx route is accessible
+$ curl my-nginx-nginx-example.apps.my-source-cluster.example.com 
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+[... snipped ...]
+
+```
+
+Next, let's fill out our 'MigPlan' specifying the `nginx-example` namespace should be moved to our destination cluster during the migration.
+
+---
+
+### 3. Create a 'MigPlan' that references our 'nginx-example' namespace
+
+To migrate our `nginx-example` namespace, we'll ensure that the `namespaces` field of our MigPlan includes nginx-example. Luckily [config/samples/mig-plan.yaml](https://github.com/fusor/mig-controller/blob/master/config/samples/mig-plan.yaml) does exactly this.
+
+```yaml
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: migplan-sample
+  namespace: mig
+
+[... snipped, see config/samples/mig-plan.yaml for full version ...]
```

Comment:
+1

goal is to ensure CI is testing as much of the scenarios as possible, so we do not want to include a full yaml in the docs, more we want to point to where the yaml is so we can be sure CI is testing and it's still accurate.

Reply:
@djwhatle Regarding this (and my comment above). I agree we shouldn't be including the full yaml in the docs, but I wonder whether we should at least mention (without including the yaml for it) that the clusters created in the earlier step should be referenced here. Otherwise it might seem like the "see elsewhere for full yaml" part is boilerplate rather than a necessary part of the config since it's an explicit object reference rather than default configuration.

Reply:
Updated the comment to hopefully be more clear that people should refer to sample to see other "required" fields.

```
[... snipped, see config/samples/mig-plan.yaml for other required fields ...]
```